.travis.yml

```diff
@@ -10,17 +10,25 @@ install:
   - conda create -p $HOME/py --yes $deps "python=$TRAVIS_PYTHON_VERSION"
   - export PATH=$HOME/py/bin:$PATH
   - pip install unittest2
+  # build our scala code
+  - ./sbt/sbt assembly
   # install our own package into the environment
   - "pip install ."
 script:
   # Download spark 1.4.1
   - "wget http://d3kbcqa49mib13.cloudfront.net/spark-1.4.1-bin-hadoop2.4.tgz"
-  - "tar -xvf spark-1.4.1-bin-hadoop2.4.tgz"
-  # build our scala code
-  - ./sbt/sbt assembly
+  - "tar -xf spark-1.4.1-bin-hadoop2.4.tgz"
   # Run the tests
   - "export SPARK_HOME=./spark-1.4.1-bin-hadoop2.4/"
-  - "export SPARK_CONF_DIR=./sparklingpandas/test/resources/conf"
-  - "nosetests --logging-level=ERROR --detailed-errors --verbosity=2 --with-coverage --cover-html-dir=./htmlcov --cover-package=sparklingpandas"
+  - "nosetests --logging-level=INFO --detailed-errors --verbosity=2 --with-coverage --cover-html-dir=./htmlcov --cover-package=sparklingpandas"
   - "pep8 --ignore=E402 sparklingpandas/"
-  - "pylint -E sparklingpandas --extension-pkg-whitelist=numpy --disable=no-member"
+  - "pylint -E sparklingpandas --extension-pkg-whitelist=numpy"
+  - echo $PATH
+  - export LOCATION=`pip show sparklingpandas|grep Location| cut -d ' ' -f 2`
+  - echo $LOCATION
+  - ls $LOCATION/../
```

Comment:
What is up with all the ls's? Why are they needed in travis? Were they just for debugging and if so can they be removed now?